source/main/gui/panels/GUI_GameSettings.cpp

```diff
@@ -386,6 +386,8 @@ void GameSettings::DrawGraphicsSettings()
     DrawGIntCheck(App::gfx_particles_mode,   _LC("GameSettings", "Enable particle gfx"));
     DrawGIntCheck(App::gfx_skidmarks_mode,   _LC("GameSettings", "Enable skidmarks"));
 
+    DrawGCheckbox(App::gfx_auto_lod,      _LC("GameSettings", "Enable OGRE's automatic mesh LOD generator (Increases loading times)"));
```

Comment:
Do we need to refer to it as "OGRE's automatic mesh LOD generator"? Could it just be "Enable automatic mesh LOD generator"?

Reply:
^